compose.yaml

```diff
@@ -74,7 +74,7 @@ services:
         <<: *spindle-prod
         image: pp-spindle-dev
         profiles: ["dev"]
-        command: flask run --host=0.0.0.0 -p 32768
```

Comment:
is this because of a new flask version?

Reply:
No, this was simply to use the extended flags (with `--`) consistently (instead of one extended flag and one abbreviated one).